chapters/05_numerical_results/03_what_if/main.tex

```diff
@@ -0,0 +1,804 @@
+each player.
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=\linewidth]{chapters/05_numerical_results/Bin/example_2/poa_ard_example_2_parking_space.pdf}
+    \caption{Example 2: Asymmetric replicator dynamics and compartmentalised
+    \(PoA\) with increased parking capacity \(M\)}
+    \label{fig:poa_ard_example_2_parking_capacity}
+\end{figure}
+
+Although this time the strategies do not change, the \(PoA\) of both players
+increases, which makes the entire game more inefficient in terms of the
+blocking time of ambulances.
+This is because the parking capacity of the hospitals is increased, which means
+that patients can now wait longer in the parking space.
+Therefore, the \(PoA\) gets worse because the blocking time of ambulances
+when playing strategies \(T^A = 5\) and \(T^B = 6\) is increased.
+
+Figures~\ref{fig:poa_ard_example_2_parking_capacity} and
+\ref{fig:poa_ard_example_2_system_capacity} demonstrate how when increasing
+the hospital capacities or the parking capacities make the outcome of the game
+even more inefficient.
+Consider now the case that the arrival rate of type 2 individuals \(\lambda_2\)
+is increased at some point during the learning algorithm.
+Similar to before, the learning algorithm is ran with the initial parameters,
+and then \(\lambda_2\) is increased.
+Figure~\ref{fig:poa_ard_example_2_lambda_2} shows the output of the asymmetric
+replicator dynamics algorithm and the compartmentalised \(PoA\) of each player.
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=\linewidth]{chapters/05_numerical_results/Bin/example_2/poa_ard_example_2_lambda_2.pdf}
+    \caption{Example 2: Asymmetric replicator dynamics and compartmentalised
+    \(PoA\) with increased arrival of type 2 patients \(\lambda_2\)}
+    \label{fig:poa_ard_example_2_lambda_2}
+\end{figure}
+
+While the strategies of the players don't change, the \(PoA\) of both players
+actually decreases.
+Increasing the arrival rate of type 2 patients (ambulance patients) shouldn't
+make the game more efficient, but it does.
+That is not because the blocking time is decreased.
+As a matter of fact the blocking time of ambulances increases for both players.
+The reason why the \(PoA\) decreases can be traced back to the definition of the
+\(PoA\) from equation~\ref{eq:compartmentalised_poa_definition}.
+\(PoA_i(s)\) is defined as ratio between the blocking time when strategy \(s\)
+is played and the best achievable blocking time from all strategies.
+Therefore, when the best achievable blocking time becomes worse, the \(PoA\) of
+the players decreases, which is exactly what happens in this case.
+Although this is an interesting outcome, it is not a sensible one.
+Increasing the arrival rate of an already flooded system does not make the
+system more efficient.
+
+A more appropriate way to increase the efficiency of the system is to increase
+the number of staff \(C\).
```

Comment:
```suggestion
the capacity \(C\).
```

Reply:
I see what you mean.

The problem with `number of staff` is that's it's too precise. In practice this could be limited by beds, limited by a combination of nurse + doctor + ...

What about `service capacity`?

Reply:
In the generic version of the model I use `number of servers`. But I don't think this could be limited by beds. I think bed availability has to do more with the artificial hospital capacity `N`, right? 🤔

I will add a `TODO` for it to revisit at a later point